core/src/main/scala/org/http4s/syntax/HeaderSyntax.scala

```diff
@@ -17,23 +17,33 @@
 package org.http4s
 package syntax
 
+import cats.data.NonEmptyList
 import org.http4s.util.Renderer
 import org.typelevel.ci.CIString
 
 trait HeaderSyntax {
   implicit def http4sHeaderSyntax[A](a: A)(implicit header: Header[A, _]): HeaderOps[A] =
     new HeaderOps(a, header)
 
-  implicit def http4sSelectSyntax[A](a: A)(implicit select: Header.Select[A]): SelectOps[A] =
-    new SelectOps(a)
+  implicit def http4sSelectSyntaxOne[A](a: A)(implicit select: Header.Select[A]): SelectOpsOne[A] =
+    new SelectOpsOne(a)
+
+  implicit def http4sSelectSyntaxMultiple[A, H[_]](a: H[A])(implicit
+      select: Header.Select.Aux[A, H]): SelectOpsMultiple[A, H] =
+    new SelectOpsMultiple[A, H](a)
 }
 
 final class HeaderOps[A](val a: A, header: Header[A, _]) {
   def value: String = header.value(a)
   def name: CIString = header.name
 }
 
-final class SelectOps[A](val a: A)(implicit ev: Header.Select[A]) {
-  def toRaw: Header.Raw = ev.toRaw(a)
+final class SelectOpsOne[A](val a: A)(implicit ev: Header.Select[A]) {
+  def toRaw: Header.Raw = ev.toRaw1(a)
```

Comment:
This seemed shady to me, but would have changed a lot to rename it to `toRaw1` where it's used.  I'm also not comfortable with how it interacts with `SelectOpsMultiple`.  We might want to take a look at this, still.

Reply:
Yeah... there could be instances of ambiguous implicits...  maybe we can rename `SelectOpsMultiple.toRaw` to something else to avoid that?

Reply:
I think the syntax needs to be consistent with the typeclass: https://github.com/zarthross/http4s/pull/2